content/src/storage/repositories/DeploymentsRepository.ts

```diff
@@ -74,19 +73,53 @@ export class DeploymentsRepository {
             offset,
         }
 
-        if (from) {
-            values.from = from
-            whereClause.push(`dep1.${timestampField} >= to_timestamp($(from) / 1000.0)`)
+        if (filters?.fromLocalTimestamp) {
+            values.fromLocalTimestamp = filters.fromLocalTimestamp
+            whereClause.push(`dep1.local_timestamp >= to_timestamp($(fromLocalTimestamp) / 1000.0)`)
         }
 
-        if (to) {
-            values.to = to
-            whereClause.push(`dep1.${timestampField} <= to_timestamp($(to) / 1000.0)`)
+        if (filters?.toLocalTimestamp) {
+            values.toLocalTimestamp = filters.toLocalTimestamp
+            whereClause.push(`dep1.local_timestamp <= to_timestamp($(toLocalTimestamp) / 1000.0)`)
         }
 
-        if (serverUrl) {
-            values.serverUrl = serverUrl
-            whereClause.push(`dep1.origin_server_url = $(serverUrl)`)
+        if (filters?.fromOriginTimestamp) {
+            values.fromOriginTimestamp = filters.fromOriginTimestamp
+            whereClause.push(`dep1.origin_timestamp >= to_timestamp($(fromOriginTimestamp) / 1000.0)`)
+        }
+
+        if (filters?.toOriginTimestamp) {
+            values.toOriginTimestamp = filters.toOriginTimestamp
+            whereClause.push(`dep1.origin_timestamp <= to_timestamp($(toOriginTimestamp) / 1000.0)`)
+        }
+
+        if (filters?.originServerUrl) {
+            values.originServerUrl = filters.originServerUrl
+            whereClause.push(`dep1.origin_server_url = $(originServerUrl)`)
+        }
+
+        if (filters?.deployedBy && filters.deployedBy.length > 0) {
+            values.deployedBy = filters.deployedBy
+            whereClause.push(`dep1.deployer_address IN ($(deployedBy:list))`)
```

Comment:
Just to be sure we are safe against SQL injection: we're just setting a query parameter here, right?